doc/contributing/docs/style.rst

```diff
@@ -174,13 +174,38 @@ Spelling and punctuation
 US vs British spelling
 ~~~~~~~~~~~~~~~~~~~~~~
 
-We use the US English spelling.
+Use the US English spelling.
 
 Check your spelling and punctuation
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 Consider checking spelling, grammar, and punctuation with special tools.
 
+Dashes
+~~~~~~
+
+Special symbols like dashes, quotation marks, and apostrophes look the same
+across all Tarantool documentation in a single language.
+This is because the documentation :doc:`builder </contributing/docs/build/>`
+renders specific character sequences in the source into correct typographic characters.
+
+Type three hyphens (``---``) to insert an em dash (---)
+and two hyphens (``--``) for an en dash (--).
+The following rules apply:
+
+*   The shorter **en dash** ``--`` is used to mark ranges (for example, 4--16 GB).
```

Comment:
Numerical ranges or ranges of any nature? As I see from the example below, the latter, but it is not stated explicitly.